barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplemini_verifier.hpp

```diff
@@ -3,7 +3,6 @@
 #include "barretenberg/commitment_schemes/commitment_key.hpp"
 #include "barretenberg/commitment_schemes/gemini/gemini.hpp"
 #include "barretenberg/commitment_schemes/shplonk/shplonk.hpp"
-#include "barretenberg/commitment_schemes/utils/batch_mul_native.hpp"
```

Comment:
The cost of including headers in headers is much higher than the cost of including headers in cpp files. He header _should_ be a description of interfaces (this sometimes breaks down/we get lazy around templates) and and the dependency structure of those should be exactly what is expressed by the inclusion of headers in headers.

Reply:
Thanks for explaining this and for the clean-up, will be more careful with the inclusions